nmma/em/create_lightcurves.py

```diff
@@ -69,8 +69,7 @@ def get_parser():
     parser.add_argument(
         "--filters",
         type=str,
-        help="A comma seperated list of filters to use (e.g. g,r,i). If none is provided, will use all the filters available",
-        default="u,g,r,i,z,y,J,H,K",
```

Comment:
Can we still give a reasonable default here?

Reply:
This will default to all the available filters of the given model, so it should be fine. But the help text should not be removed but updated.

Reply:
Sounds good.